cogs/make_member.py

```diff
@@ -153,16 +151,21 @@ async def make_member(self, ctx: TeXBotApplicationContext, group_member_id: str)
 
         guild_member_ids: set[str] = set()
 
-        request_headers: dict[str, str] = {
+        REQUEST_HEADERS: dict[str, str] = {
```

Comment:
Unnecessary capitalisation. The request headers should be allowed to be edited between initial declaration and usage.

Reply:
I did this manually to be more consistent with elsewhere - such as in the get_token_authorisation command

Reply:
Hmmmm. I suppose I agree that they should be constants, however in that case they should be put outside the class & type hinted as `Final[Mapping[str, str]]` to prevent editing.

Could you also update the ones in the `get_token_authorisation()` command for consistency, within this PR?